bindings/C/c/adios2_c_internal.inl

```diff
@@ -15,6 +15,8 @@
 #error "Inline file should only be included from its header, never on its own"
 #endif
 
+#include <complex>
```

Comment:
see comment above, this is a C++ header in a C exposed header

Reply:
See above, it's not an exposed header

Reply:
fair enough